high_noise_filter: 0.0 # secs
